File: 1.Strategy-Pattern/Java/src/MiniDuckSimulator.java
import Ducks.Duck;
import Ducks.MallardDuck;
import Ducks.ModelDuck;
import FlyBehaviors.FlyRocketPowerd;

public class MiniDuckSimulator {
    public static void main (String[] args) {
        Duck mallard = new MallardDuck();
        mallard.performFly();
        mallard.performQuack();

        Duck model = new ModelDuck();
        model.performFly();

        model.setFlyBehavior(new FlyRocketPowerd());
        model.performFly();
    }
}

File: 1.Strategy-Pattern/PHP/MiniDuckSimulator.php
<?php

require 'vendor/autoload.php';

$mallard = new MallardDuck();
$mallard->performQuack();
$mallard->performFly();

$model = new ModelDuck();
$model->performQuack();
$model->performFly();

$model->setFlyBehavior(new FlyWithWings());
$model->performFly();